packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -97,7 +97,72 @@ const mentionTag = `
   Displayable Text
 </msft-mention>
 `;
-const getDocs: () => JSX.Element = () => {
+const Docs: () => JSX.Element = () => {
+  const refDefaultMessageThread = useRef(null);
+  const refWithMessageDate = useRef(null);
+  const refSystemMessage = useRef(null);
+  const refBlockedMessage = useRef(null);
+  const refCustomMessage = useRef(null);
+  const refWithCustomizedChatContainer = useRef(null);
+  const refWithCustomizedMessageContainer = useRef(null);
+  const refWithCustomizedBlockedMessageContainer = useRef(null);
+  const refDefaultMessageWithStatusIndicator = useRef(null);
+  const refCustomMessageWithStatusIndicator = useRef(null);
+  const refCustomAvatar = useRef(null);
+  const refCustomTimestamp = useRef(null);
+  const refDisplayInlineImages = useRef(null);
+  const refDisplayFileAttachments = useRef(null);
+  const refMentionOfUsers = useRef(null);
+  const refProps = useRef(null);
+
+  const scrollToRef = (ref): void => {
+    setTimeout(() => {
+      ref.current.scrollIntoView({ behavior: 'auto' });
+    }, 1000);
+  };
+
+  useEffect(() => {
+    const url = document.URL;
```

Comment:
The scroll issue in Edge and Chrome might stem from the URL not being retrieved correctly on this page. 
For instance, upon entering the URL `http://localhost:6006/?path=/docs/ui-components-messagethread--message-thread#display-file-attachments-with-messages`, the `document.URL` yields "http://localhost:6006/iframe.html?viewMode=docs&id=ui-components-messagethread--message-thread" instead.

By updating from `const url = document.URL;` to `const url = window.top.location.href;`, the functionality is restored in Edge and Chrome. However, this might introduce url fetching failure case in my memory. It would be worthy to confirm with Web expert how to achieve the URL path